src/types.rs

```diff
@@ -54,10 +62,30 @@ pub struct ExtremePoint<T>
     pub xmin: Point<T>,
 }
 
+/// A single Point in 2D space.
+///
+/// Points can be created using the `new(x, y)` constructor, or from a `Coordinate` or pair of points.
+///
+/// ```
```

Comment:
w.r.t. `# Examples`, most of the existing doc comments don't have it as a separate heading, so for consistency reasons I didn't put it in here. :)